### DISCUSSION TITLE
Whole series of recent Dell monitors don't support DDC

### DISCUSSION THREAD

**Discussion by runevision** [MonitorControl/MonitorControl#1162] [General]
I'm writing here because this took me a long time to figure out and I hope to raise awareness about this.

The about page says:
"Most modern LCD displays from all major manufacturers supported implemented DDC/CI protocol via DisplayPort, HDMI, USB-C or VGA to allow for hardware backlight control."

However, most certainly don't mean all, and some monitors that say in their specs they support DDC don't actually support it on Mac. This is important to be aware of! Just because the specs page of a monitor says it supports DDC don't mean you can count on that if you're a Mac user. This is relevant to people using MonitorControl.

Case in point. Dell's P series of monitors do not support DDC on Mac at all. Specs pages for P series monitors, such as the P2720D monitor I bought (which is from 2020), says that DDC is supported. And there's also a setting in the on-screen-display menu for turning DDC on/off. But on Mac it's not supported. You can see here that Dell's software for controlling monitors don't support the P series of monitors at all, only the U, UP and C series are supported:
https://www.dell.com/support/kbdoc/en-ca/000201067/dell-display-and-peripheral-manager-for-macos

When I use MonitorControl set to DCC, moving the slider for my Dell monitor does nothing at all (except in the lower part of the slider that does software adjustment). Changing it to use gamma makes it work, but that's inferior given it doesn't adjust the monitor backlight at all. I also tried the program called Lunar, and it too reports that the screen doesn't support DDC.

How can we spread more awareness that for Mac users, it's not all modern monitors that support DDC, and even looking at the spec pages for a given monitor is not enough, since it doesn't guarantee support on Mac?

I would suggest tweaking the wording on the about page of the quoted text above, to make it more clear one needs to be vigilant in one's research when buying a monitor for use with Mac.

**Comment by waydabber** [comment]
Hi there, are you sure you are not using the HDMI port (which does not work on 2018 and newer Macs for DDC #750)?

**Reply by runevision** [reply]
Aha! Yes I was using the HDMI port. I now tried to use one of the USBC/thunderbolt ports with a HDMI adapter instead, and it turns out DDC then does work (as you suspected), contrary to Dell's own information about DDC support on Mac. This mess of misleading information is really confusing to navigate.

I'm happy to learn this means I can software adjust the backlight of my external display after all, even if it's also disappointing to learn that Apple shipped their MacBook Pros with new HDMI ports that are lacking basic functionality so they're no use anyway.

Thanks for the help!
